core/sql/executor/ExHbaseAccess.cpp

```diff
@@ -2996,6 +3003,14 @@ void ExHbaseAccessTcb::handleException(NAHeap *heap,
      errorMsgLen = strlen("[UNKNOWN EXCEPTION]\n");
```

Comment:
Looks like errorMsg is uninitialized in this code path. Perhaps we need to add, "errorMsg = "[UNKNOWN EXCEPTION]" here? (And maybe change this line to errorMsgLen = strlen(errorMsg)?)